tests/test_repo_commands.py

```diff
@@ -27,6 +27,12 @@ def test_init():
     assert result.exit_code != 0
 
 
+def test_init_nohooks():
```

Comment:
Coverage decreased by about 2.7%, should look at the output of

`py.test --cov-report html --cov=zazu tests/` and see whats missing.